src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: Figure['shape'] = 'triangle';
```

Comment:
And after adding enum you can simply select the desired key in ename, which will return the value to you
`shape = Shape.Triangle;`